internal/pkg/model/manifest.go

```diff
@@ -117,6 +118,10 @@ func (p *Paths) AddRelatedPath(path string) {
 	p.RelatedPaths = append(p.RelatedPaths, relPath)
 }
 
+func (p *Paths) AddRelatedPathInRoot(path string) {
+	p.RelatedPaths = append(p.RelatedPaths, fmt.Sprintf("..%c%s", filesystem.PathSeparator, path))
+}
```

Comment:
No nevím co mi na to řekneš, ale připadalo mi to takto relativně nejčistší jak obejít kontrolu a skládání cesty v `Paths.AddRelativePath()`. 🙂

Reply:
Ok, nejaky hack tam byt musi, toto mi pride v pohode.